ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/ParticipantConfig.scala

```diff
@@ -29,4 +31,7 @@ object ParticipantConfig {
     s"jdbc:h2:mem:$participantId;db_close_delay=-1;db_close_on_exit=false"
 
   val defaultManagementServiceTimeout: Duration = Duration.ofMinutes(2)
+
+  val defaultIndexerDatabaseConnectionPoolSize = 10
+  val defaultApiServerDatabaseConnectionPoolSize = 50
```

Comment:
These seem high. Could you justify them in comments, please?

Reply:
I lowered the defaults and added comments.